frontend/packages/component-library/src/formFieldWrapper/__tests__/FormFieldWrapper.test.tsx

```diff
@@ -0,0 +1,96 @@
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import FormFieldWrapper, {FormFieldWrapperProps} from '../FormFieldWrapper';
+
+describe('Design System - FormFieldWrapper', () => {
+  const renderFormFieldWrapper = (
+    props: Partial<FormFieldWrapperProps> = {},
+  ) => {
+    render(<FormFieldWrapper {...props} />);
+  };
+
+  const queryFormFieldWrapperEl = () =>
+    document.querySelector('.formFieldWrapper');
```

Comment:
non blocking, may be a follow up, but I wonder if we can use storybook play function to test the functionality that is currently tested via this function here. (in storybook we can actually see/check the visuals, in jest we can only check classNames, which we try to avoid)